### PR TITLE
Activates widget avg/max/min within the editor

### PR DESCRIPTION
**Pull request by fdansv**
Fixes https://github.com/CartoDB/deep-insights.js/issues/415

@xavijam @piensaenpixel

### CODE REVIEW

**Comment by xavijam** [comment]
This will only work in the first iteration, but if a new widget is added it will not work. In my opinion this should not be there, should be in the widget creation.

File: lib/assets/javascripts/cartodb3/deep-insights-integrations.js [hunk 1]
```diff
@@ -47,10 +47,10 @@ var F = function (opts) {
   opts.widgetDefinitionsCollection.on('add remove reset', this._invalidateSize, this);
 
   this.widgetDefinitionsCollection = opts.widgetDefinitionsCollection;
-  this.activateWidgetsStats();
 
   this._analysisDefinitionsCollection.each(this._analyseDefinition, this);
-  this._vis().on('reload', this._recordChange, this);
+  this._vis().on('reload', this._visReload, this);
+  this._vis().on('change:error', this._visErrorChange, this);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/deep-insights-integrations.js]: What is that? Is related with this PR?

Comment (fdansv) [inline lib/assets/javascripts/cartodb3/deep-insights-integrations.js]: no, that's a bad merge I just added by accident, reverting

**Comment by xavijam** [comment]
This is already working. 💋

**Pull request closed without merging**